float32 posX
float32 posY
float32 orientation
float32 size
---
